guidelines: unify most of the quotations

ArturJD96 opened the pull request:
Cleaning docks from superfluous quotations marks. Adding more specific tags.

musicEnfanthen committed:
fix: update meiversion to next dev version

musicEnfanthen committed:
ci: add validation for PRs

musicEnfanthen committed:
ci: add mininmal default permissions in deploy workflow

musicEnfanthen committed:
ci: update permissions, refs and name

musicEnfanthen committed:
ci: allow reading PRs only on job level

musicEnfanthen committed:
fix: stick with two-dgit version number for better comprehensibility

bwbohl committed:
Update meilogo_ascii_100.txt

bwbohl committed:
Update source/mei-source.xml

Co-authored-by: Stefan Münnich <stefan.muennich@unibas.ch>

krHERO committed:
add missing link

Refs #https://github.com/music-encoding/music-encoding/issues/1488

krHERO committed:
add specList for front and back to chapter 9

Refs https://github.com/music-encoding/music-encoding/issues/1487

Resolved conflict.

krHERO committed:
correct misleading filename, id and title of chapter 12

Refs https://github.com/music-encoding/music-encoding/issues/1490

musicEnfanthen committed:
fix: fix version number in edition

krHERO committed:
update links

Commit:
Changed apostrophes to consistent glyph. Removed most of the quotation marks (replaced with 'title', 'quote', 'q', 'hi' or 'val' tags).

rettinghaus commented on source/docs/01-introduction.xml:
```diff
@@ -478,28 +478,28 @@
                      <item>The logical domain is the basic musical content – the essence from which
                         all performances and editions of the work are derived, including virtual
-                        time values, nominal pitches, etc. The logical domain is describable as “the
+                        time values, nominal pitches, etc. The logical domain is describable as <q?the
```
```suggestion
                        time values, nominal pitches, etc. The logical domain is describable as <q>the
```

rettinghaus commented on source/docs/02-shared.xml:
```diff
@@ -214,15 +214,15 @@
                         </figure>
                      </p>
-                     <p>While dramatic works, such as Verdi's opera, <hi rend="italic">Il Trovatore</hi>, often exhibit a more deeply-nested structure:</p>
+                     <p>While dramatic works, such as Verdi’s opera, <hi rend="italic">Il Trovatore</hi>, often exhibit a more deeply-nested structure:</p>
```
```suggestion
                     <p>While dramatic works, such as <name ref="https://en.wikipedia.org/wiki/Giuseppe_Verdi">Verdi’s</name> opera, <hi rend="italic">Il Trovatore</hi>, often exhibit a more deeply-nested structure:</p>
```

rettinghaus commented on source/docs/03-metadata.xml:
```diff
@@ -304,12 +304,12 @@
                      <p>Clearly, there will always be borderline cases and the matter is somewhat arbitrary. The simplest rule is: if you think that your file is a new edition, then call it such. An edition statement is optional for the first release of a computer file; it is mandatory for each later release, though this requirement cannot be enforced.</p>
                      <p>Note that all changes in a file, whether or not they are regarded as constituting a new edition or simply a revision, should be independently noted in the revision description section of the file header (see section <ptr target="#headerRevisionDescription"/>).</p>
-                     <p>The edition element should contain phrases describing the edition or version, including the word 'edition', 'version', or an equivalent term, together with a number or date, or terms indicating difference from other editions such as 'new edition', 'revised edition', etc. Any dates that occur within the edition statement should be marked with the <gi scheme="MEI">date</gi>  ...
+                     <p>The edition element should contain phrases describing the edition or version, including the word <val>edition</val>, <val>version</val>, or an equivalent term, together with a number or date, or terms indicating difference from other editions such as <q>new edition</q>, <q>revised edition</q>, etc. Any dates that occur within the edition statement should be marked with the ...
```
I don't think using `val` is correct here.

ArturJD96 replied:
Changing to `<q>`

rettinghaus commented on source/docs/03-metadata.xml:
```diff
@@ -333,10 +333,10 @@
                      <p>The <gi scheme="MEI">extent</gi> element describes the approximate size of a text as stored on some carrier medium, whether digital or non-digital, specified in any convenient units.</p>
                      <p>For printed books, information about the carrier, such as the kind of medium used and its size, are of great importance in cataloging procedures. The print-oriented rules for bibliographic description of an item’s medium and extent need some re-interpretation when applied to electronic media. An electronic file exists as a distinct entity quite independently of its carrier ...
-                     <p>This is particularly true of information about file-type although library-oriented rules for cataloging often distinguish two types of computer file: ‘data’ and ‘programs’. This distinction is quite difficult to draw in some cases, for example, hypermedia or texts with built-in search and retrieval software.</p>
+                     <p>This is particularly true of information about file-type although library-oriented rules for cataloging often distinguish two types of computer file: <val>data</val> and <val>programs</val>. This distinction is quite difficult to draw in some cases, for example, hypermedia or texts with built-in search and retrieval software.</p>
```
Again, `val` might not be the correct tag.

ArturJD96 replied:
Changing to `<hi>`

rettinghaus commented on source/docs/03-metadata.xml:
```diff
@@ -445,7 +445,7 @@
                         </figure>
                      </p>
-                     <p>The <gi scheme="MEI">identifier</gi> element may be used to supply any identifying number associated with the series, including both standard numbers such as an ISSN and particular issue numbers. Its <att>type</att> attribute is used to categorize the number further, taking the value 'ISSN' for an ISSN, for example.</p>
+                     <p>The <gi scheme="MEI">identifier</gi> element may be used to supply any identifying number associated with the series, including both standard numbers such as an ISSN and particular issue numbers. Its <att>type</att> attribute is used to categorize the number further, taking the value <abbr>ISSN</abbr> for an ISSN, for example.</p>
```
```suggestion
                     <p>The <gi scheme="MEI">identifier</gi> element may be used to supply any identifying number associated with the series, including both standard numbers such as an <abbr>ISSN</abbr> and particular issue numbers. Its <att>type</att> attribute is used to categorize the number further, taking the value <val>ISSN</val> for an <abbr>ISSN</abbr>, for example.</p>
```

rettinghaus commented on source/docs/03-metadata.xml:
```diff
@@ -486,16 +486,16 @@
                      <list rend="bulleted">
                         <item>the nature, scope, artistic form, or purpose of the work; also the genre or other intellectual category to which it may belong. These should be formally described within the <gi scheme="MEI">workList</gi> element (section <ptr target="#headerWorkDescription"/>).</item>
-                        <item>bibliographic details relating to the source or sources of an electronic text: <abbr>e.g.</abbr>, ‘Transcribed from a facsimile of the 1743 publication’. These should be formally described in the <gi scheme="MEI">sourceDesc</gi> element (section <ptr target="#headerSourceDescription"/>).</item>
+                        <item>bibliographic details relating to the source or sources of an electronic text: <abbr>e.g.</abbr>, <q>Transcribed from a facsimile of the 1743 publication<q>. These should be formally described in the <gi scheme="MEI">sourceDesc</gi> element (section <ptr target="#headerSourceDescription"/>).</item>
```
```suggestion
                        <item>bibliographic details relating to the source or sources of an electronic text: <abbr>e.g.</abbr>, <q>Transcribed from a facsimile of the 1743 publication</q>. These should be formally described in the <gi scheme="MEI">sourceDesc</gi> element (section <ptr target="#headerSourceDescription"/>).</item>
```

rettinghaus commented on source/docs/03-metadata.xml:
```diff
@@ -1466,7 +1466,7 @@
                         <specDesc key="extent"/>
                      </specList></p>
-                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in units such as ‘number of pages’, ‘number of folios’, ‘records’, ‘bytes’, ‘physical components’, etc. For specifying the physical dimensions of the material—for example, height and width—the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
+                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in units such as <val>number of pages</val>, <val>number of folios</val>, <val>records</val>, <val>bytes</val>, <val>physical components</val>, etc. For specifying the physical dimensions of the material—for example, height and width—the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
```
Also here, not exactly "values".

ArturJD96 replied:
Changed to `<q>`

rettinghaus commented on source/docs/11-scholarlyediting.xml:
```diff
@@ -437,7 +437,7 @@
                         </figure>
                      </p>
-                     <p>When the presumed loss of text arises from an identifiable cause, <att>agent</att> signifies the causative agent. When the presumed loss of text arises from action (partial deletion, etc.) assignable to an identifiable hand, the <att>hand</att> attribute signifies the hand responsible for the action. The <att>reason</att> attribute indicates why the text has to be supplied ...
+                     <p>When the presumed loss of text arises from an identifiable cause, <att>agent</att> signifies the causative agent. When the presumed loss of text arises from action (partial deletion, etc.) assignable to an identifiable hand, the <att>hand</att> attribute signifies the hand responsible for the action. The <att>reason</att> attribute indicates why the text has to be supplied ...
```
```suggestion
                     <p>When the presumed loss of text arises from an identifiable cause, <att>agent</att> signifies the causative agent. When the presumed loss of text arises from action (partial deletion, etc.) assignable to an identifiable hand, the <att>hand</att> attribute signifies the hand responsible for the action. The <att>reason</att> attribute indicates why the text has to be supplied, <abbr>e.g.</abbr>, <val>overbinding</val>, <val>faded ink</val>, <val>lost folio</val>, <val>omitted in original</val>, etc. The <att>source</att> attribute contains the source of the supplied text. The editor(s) responsible for supplied material may be recorded in the <att>resp</att> attribute. The value of <att>resp</att> must point to one or more identifiers declared in the document header. The <att>cert</att> attribute signifies the degree of certainty ascribed to the supplied material.</p>
```

rettinghaus commented on source/docs/11-scholarlyediting.xml:
```diff
@@ -500,7 +500,7 @@
                      <p>The <att>desc</att> attribute gives a prose description of the means of restoration. The <att>cert</att> attribute signifies the degree of certainty ascribed to the identification of the hand of the restoration. The <att>type</att> attribute may be used to indicate the action cancelled by the restoration. The <att>resp</att> attribute contains an ID reference to an element ...
                      <p>MEI offers a <gi scheme="MEI">handShift</gi> milestone element that can be used to mark a change of scribe or scribal style.</p>
-                     <p>The <att>character</att> attribute describes characteristics of the hand, particularly those related to the quality of the writing, <abbr>e.g.</abbr>, 'shaky', 'thick', regular'. A description of the tint or type of ink, <abbr>e.g.</abbr>, 'brown' or the writing medium, <abbr>e.g.</abbr>, 'pencil', may be placed in the <att>medium</att> attribute.</p>
+                     <p>The <att>character</att> attribute describes characteristics of the hand, particularly those related to the quality of the writing, <abbr>e.g.</abbr>, <var>shaky</var>, <var>thick</var>, <var>regular</var>. A description of the tint or type of ink, <abbr>e.g.</abbr>, <var>brown</var> or the writing medium, <abbr>e.g.</abbr>, <var>pencil</var>, may be placed in the <att>med ...
```
```suggestion
                     <p>The <att>character</att> attribute describes characteristics of the hand, particularly those related to the quality of the writing, <abbr>e.g.</abbr>, <var>shaky</var>, <var>thick</var>, <val>regular</val>. A description of the tint or type of ink, <abbr>e.g.</abbr>, <val>brown</val> or the writing medium, <abbr>e.g.</abbr>, <val>pencil</val>, may be placed in the <att>medium</att> attribute.</p>
```

ArturJD96 committed:
Update source/docs/01-introduction.xml

Corrected typo ("<q>", not "<q?").

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

ArturJD96 committed:
Update source/docs/02-shared.xml

Added <name> tag around Verdi.

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

ArturJD96 committed:
Update source/docs/03-metadata.xml

Added <abbr> around ISSN. Note: perhaps more to find...

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

ArturJD96 committed:
Update source/docs/11-scholarlyediting.xml

Changed incorrect <var> to <val> (perhaps to legacy JS overdose).

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

ArturJD96 committed:
Update source/docs/11-scholarlyediting.xml

Changed incorrect <var> to <val>

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

ArturJD96 committed:
Update source/docs/03-metadata.xml

Added missing `/`.

Co-authored-by: Klaus Rettinghaus <klaus.rettinghaus@gmail.com>

Commit:
Corrected misused val tags.

ArturJD96 committed:
Merge branch 'develop' into quotation-marks-1498

Commit:
Corrected typo.

Commit:
Removed wrong <hi>, converted more apostrophes to <val>.

Commit:
Corrected formating.

Commit:
Fixed type in </val>

Commit:
Closed author tag

Commit:
Changed (most of the) <hi> tags into <hi rend=italic

Commit:
Fixed double rend rend on a <hi> tag.

musicEnfanthen commented on .
:
@ArturJD96 Please remove this file from the PR. Seems to be commited by accident

Commit:
Removed 'dot' file (seemed to be pushed by mistake).

ArturJD96 committed:
Merge branch 'develop' into quotation-marks-1498

ArturJD96 commented on .
:
Removed!

bwbohl commented on source/docs/13-linkingdata.xml:
```diff
@@ -96,10 +96,10 @@
          <div xml:id="linkingdata" type="div1">
             <head>Linking Data</head>
-            <p>This chapter describes the use of elements in MEI for linking and referencing. This includes the elements, models, and attributes that are part of the 'MEI.ptrref' module. This module contains declarations, techniques and approaches to establish references within a single MEI document, or to link out from one MEI document to another or to other external sources. This chapter also a ...
+            <p>This chapter describes the use of elements in MEI for linking and referencing. This includes the elements, models, and attributes that are part of the <hi>MEI.ptrref</hi> module. This module contains declarations, techniques and approaches to establish references within a single MEI document, or to link out from one MEI document to another or to other external sources. This chapter ...
```
A `tei:hi` without a `@rend` doesn’t do the job. @rettinghaus, maybe we could have a session on figuring out a more concise schema and encoding guidelines for our documentation. What do you think?

ArturJD96 replied:
There are many `@rend`erless `tei:hi` in the header like `<hi>Music Encoding Initiative (MEI)</hi>`. This one, as well as the one you highlighted I left because it is ambiguous what to. I appended `@rend` for most of the `<hi>` in [8bdc46a](https://github.com/music-encoding/music-encoding/pull/1626/commits/8bdc46ac6135d340404792ea6d9650c2578b2068).

ArturJD96 replied:
@rettinghaus, thank you for your feedback! I shall update it using `emph` and `term`.
I see that `emph` accepts not only `rend` but also `font-style` and `rendition`.  Would `emph` without any attribute pointing to "italic" differ to using `hi`?

rettinghaus replied:
@ArturJD96 The xlst for generating the Guidelines doesn't react to any attributes, `tei:emph` is just transformed into `html:em`.

Commit:
Added <abbr> when mentioning MEI. Added few <institution>. Added <licence> inside <availability>.

Commit:
Added <abbr> when mentioning TEI in chapter 01.

Commit:
Added <abbr> when mentioning XML or RNG in ch.1.

Commit:
Changed most of the <rend> into <emph> or <term> (unless the foreign-language vocab is used).

Commit:
Add <abbr> when ODD is mentioned in ch1.

ArturJD96 committed:
Merge branch 'music-encoding:develop' into quotation-marks-1498

Commit:
Corrected missed <hi>.

Commit:
Added missing </p>.

Commit:
Removed redundant </title>.

ArturJD96 commented:
# Updating docs further
Dear MEI colleagues,
I have corrected the first chapter according to @rettinghaus suggestions. I also added `<abbr>` whenever possible. My questions are:
1) should I also add `<expan>` where the term is explained (e.g. `<abbr>MEI</abbr> means <expan>Music Encoding Initiative</expan>`)? I checked the examples in TEI docs but they never mention anything about `<expan>` outside `<choice>` (I assume displaying context).
2) I left `<hi rend="italic">` when a foreign word is mentioned (e.g. <hi rend="italic">forte</hi>). Is it the right approach or would you prefer another tag for this situation?
3) I also added `<license>` tag to the document description. Are those deeper interventions welcome? 

If you agree, I can do the remaining docs in a similar spirit.

rettinghaus commented on source/docs/01-introduction.xml:
```diff
@@ -131,11 +133,11 @@
                   <p>Grateful acknowledgment is given to the following institutions for their
                      generous contributions: the Akademie der Wissenschaften und der Literatur (AdW)
-                     in Mainz for serving as hosting institution for the MEI Community, and the
+                     in Mainz for serving as hosting institution for the <abbr>MEI</abbr>Community, and the
```
Watch out for whitespace where it should be kept!

ArturJD96 replied:
[Fixed](https://github.com/music-encoding/music-encoding/pull/1626/commits/39e57bbb8d6e3edd8d5a37d10a6c23c5bb9e22e6).

rettinghaus commented on source/docs/01-introduction.xml:
```diff
@@ -26,11 +26,11 @@
          <titleStmt>
             <title level="s">The Music Encoding Initiative Guidelines</title>
-            <title level="a">Introduction to MEI</title>
+            <title level="a">Introduction to <abbr>MEI</abbr></title>
```
Personally I wouldn't add `abbr` tags within the header, but that should be decided by the general editors.

bwbohl replied:
@music-encoding/guidelines-editors-in-chief opinions?

pe-ro replied:
I'm all for making the Guidelines consistent, using the same markup for quotations, for example.  But marking every abbreviation, providing a corresponding expansion, and so on, runs into the law of diminishing returns and introduces the possibility of new confusion.

bwbohl replied:
I wouldn’t either, after all it’s the `fileDesc` we’re talking about.

Commit:
Fixed lacking spaces.

bwbohl commented on source/docs/01-introduction.xml:
```diff
@@ -65,24 +65,26 @@
          </titleStmt>
          <publicationStmt>
-            <distributor>Music Encoding Initiative (MEI) Board</distributor>
+            <distributor><institution>Music Encoding Initiative (<abbr>MEI</abbr>) Board</institution></distributor>
```
I think, also here, the `<distributor>` should already be sufficient

bwbohl replied:
Moreover, `institution` is not valid against our mei_odds.rng

martha-thomae commented:
Just for documentation purposes, I report that during the last ODD Meeting, it was discussed that _Kristin_ is looking into this, trying to address first only the issue of the quotation marks. And then each invidual feature involved in this PR can be addressed later through invidual PRs.

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -65,24 +65,26 @@
             <availability>
                <p>
-                  <hi>Music Encoding Initiative (MEI)</hi>
+                  <hi>Music Encoding Initiative (<abbr>MEI</abbr>)</hi>
```
```suggestion
                  <hi>Music Encoding Initiative (MEI)</hi>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -65,24 +65,26 @@
+                <p>This is a derivative work based on earlier versions of the schema copyright (c)
+                    2001-2006 Perry Roland and the Rector and Visitors of the <institution>University of Virginia</institution>;
+                    licensed under the Educational Community License version 1.0.</p>
+               </licence>
```
```suggestion
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
          <div xml:id="introduction" type="div1">
-            <head>Introduction to MEI</head>
-            <p>Welcome to the MEI Guidelines. They provide documentation for the Music Encoding
+            <head>Introduction to <abbr>MEI</abbr></head>
```
```suggestion
            <head>Introduction to MEI</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
             <div xml:id="about" type="div2">
                <head>About these Guidelines</head>
-               <p>The MEI Guidelines are intended to serve as a reference tool for music encoders.
+               <p>The <abbr>MEI</abbr> Guidelines are intended to serve as a reference tool for music encoders.
```
```suggestion
               <p>The MEI Guidelines are intended to serve as a reference tool for music encoders.
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
-               translating XML and RNG terminology and concepts into more accessible language, it is
-               still a technical one that presupposes a minimal understanding of XML and music
-               notation. Novice encoders may want to start their MEI experience by doing an <ref
+               assists users of <abbr>MEI</abbr> in achieving effective and consistent markup. Despite
```
```suggestion
               assists users of MEI in achieving effective and consistent markup. Despite
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
-               still a technical one that presupposes a minimal understanding of XML and music
-               notation. Novice encoders may want to start their MEI experience by doing an <ref
+               assists users of <abbr>MEI</abbr> in achieving effective and consistent markup. Despite
+               translating <abbr>XML</abbr> and <abbr>RNG</abbr> terminology and concepts into more accessible language, it is
```
```suggestion
               translating XML and RNG terminology and concepts into more accessible language, it is
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
-               complete. MEI is used in various contexts, and not every use-case may be fully
-               reflected in these Guidelines. However, MEI is a community effort, so feedback and
+               complete. <abbr>MEI</abbr> is used in various contexts, and not every use-case may be fully
+               reflected in these Guidelines. However, <abbr>MEI</abbr> is a community effort, so feedback and
```
```suggestion
               reflected in these Guidelines. However, MEI is a community effort, so feedback and
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -93,28 +95,28 @@
-               touch with the MEI community can be found on the <ref
-                  target="https://music-encoding.org/community/community-contacts.html">MEI
+               touch with the <abbr>MEI</abbr> community can be found on the <ref
+                  target="https://music-encoding.org/community/community-contacts.html"><abbr>MEI</abbr>
```
```suggestion
                  target="https://music-encoding.org/community/community-contacts.html">MEI
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -131,11 +133,11 @@
+                     in Mainz for serving as hosting institution for the <abbr>MEI</abbr> Community, and the
                      National Endowment for the Humanities (NEH) and the Deutsche
-                     Forschungsgemeinschaft (DFG) for their joint financial support of the MEI
+                     Forschungsgemeinschaft (DFG) for their joint financial support of the <abbr>MEI</abbr>
```
```suggestion
                     Forschungsgemeinschaft (DFG) for their joint financial support of the MEI
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -131,11 +133,11 @@
+                     Forschungsgemeinschaft (DFG) for their joint financial support of the <abbr>MEI</abbr>
                      project in its early stages. We thank several institutions that hosted Music
-                     Encoding Conferences or other MEI-related meetings in the past: The AdW Mainz,
+                     Encoding Conferences or other <abbr>MEI</abbr>-related meetings in the past: The AdW Mainz,
```
```suggestion
                     Encoding Conferences or other MEI-related meetings in the past: The AdW Mainz,
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -144,16 +146,16 @@
                      Salzburg. We also thank all other institutions that allow their researchers to
                      invest time into both the community and the encoding framework. It is their
-                     interest that makes MEI an incredible platform for interchange and scholarly
+                     interest that makes <abbr>MEI</abbr> an incredible platform for interchange and scholarly
```
```suggestion
                     interest that makes MEI an incredible platform for interchange and scholarly
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -144,16 +146,16 @@
-                     addition to providing much of the inspiration for MEI, the TEI organization
-                     supplied funding for the MEI Technical Group in its efforts to adopt ODD. The
-                     editors of these Guidelines are grateful for those of the TEI, which provided a
+                     addition to providing much of the inspiration for <abbr>MEI</abbr>, the <abbr>TEI</abbr> organization
```
```suggestion
                     addition to providing much of the inspiration for MEI, the TEI organization
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -144,16 +146,16 @@
+                  <p><abbr>MEI</abbr> has been a community-driven effort for more than a decade, and many
                      individuals have provided significant and much-appreciated commitments of time
-                     and energy to the development of MEI: Nikolaos Beer; Vincent Besson; Benjamin
+                     and energy to the development of <abbr>MEI</abbr>: Nikolaos Beer; Vincent Besson; Benjamin
```
```suggestion
                     and energy to the development of MEI: Nikolaos Beer; Vincent Besson; Benjamin
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -168,33 +170,33 @@
                <div xml:id="aboutVersion5_1" type="div3">
                   <head>About version 5.1</head>
-                  <p>Following release 5.0 from September 2023, Release 5.1 of MEI focuses primarily on maintenance 
+                  <p>Following release 5.0 from September 2023, Release 5.1 of <abbr>MEI</abbr> focuses primarily on maintenance 
```
```suggestion
                  <p>Following release 5.0 from September 2023, Release 5.1 of MEI focuses primarily on maintenance 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -168,33 +170,33 @@
-                     The Release Managers for MEI 5.1 were the Technical Co-Chairs, Benjamin W. Bohl and Stefan Münnich.</p>
-                  <p>With MEI 5.1, five additional elements have been added to the 
+                     are <emph>not</emph> displayed when its value is set to <val>true</val>. 
+                     The Release Managers for <abbr>MEI</abbr> 5.1 were the Technical Co-Chairs, Benjamin W. Bohl and Stefan Münnich.</p>
```
```suggestion
                     The Release Managers for MEI 5.1 were the Technical Co-Chairs, Benjamin W. Bohl and Stefan Münnich.</p>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                <div xml:id="aboutVersion5" type="div3">
                   <head>About version 5.0</head>
-                  <p>Release 5.0 of MEI focuses primarily on the guidelines, development infrastructure, 
+                  <p>Release 5.0 of <abbr>MEI</abbr> focuses primarily on the guidelines, development infrastructure, 
```
```suggestion
                  <p>Release 5.0 of MEI focuses primarily on the guidelines, development infrastructure, 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                   <div xml:id="meiBasic" type="div4">
-                     <head>MEI Basic</head>
-                     <p>As a framework to encode music, MEI offers extensive flexibility to encode music
+                     <head><abbr>MEI</abbr> Basic</head>
```
```suggestion
                     <head>MEI Basic</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
+                     of <abbr>MEI</abbr> as a notation interchange format. For developers, providing "full" <abbr>MEI</abbr> support 
                      is a difficult and time-consuming chore, writing and supporting code for features 
-                     which most of their users will not use. Accordingly, MEI has not seen a great deal of adoption 
+                     which most of their users will not use. Accordingly, <abbr>MEI</abbr> has not seen a great deal of adoption 
```
```suggestion
                     which most of their users will not use. Accordingly, MEI has not seen a great deal of adoption 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                      by current score-writing applications.</p>
-                     <p>This is addressed this with the release of MEI 5. We are now offering a new customization of MEI, 
-                     <hi rend="italic">MEI Basic</hi>, that provides a simplified subset of the MEI framework that 
+                     <p>This is addressed this with the release of <abbr>MEI</abbr> 5. We are now offering a new customization of <abbr>MEI</abbr>, 
```
```suggestion
                     <p>This is addressed this with the release of MEI 5. We are now offering a new customization of MEI, 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
-                     <p>This is addressed this with the release of MEI 5. We are now offering a new customization of MEI, 
-                     <hi rend="italic">MEI Basic</hi>, that provides a simplified subset of the MEI framework that 
+                     <p>This is addressed this with the release of <abbr>MEI</abbr> 5. We are now offering a new customization of <abbr>MEI</abbr>, 
+                     <emph><abbr>MEI</abbr> Basic</emph>, that provides a simplified subset of the <abbr>MEI</abbr> framework that 
```
```suggestion
                     <emph>MEI Basic</emph>, that provides a simplified subset of the MEI framework that 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                      reflects the capabilities of most popular "Common Western Music Notation" score-writing 
                      applications currently in use.</p>
-                     <p>In the full MEI schema there are often multiple ways to encode something. MEI Basic 
+                     <p>In the full <abbr>MEI</abbr> schema there are often multiple ways to encode something. <abbr>MEI</abbr> Basic 
```
```suggestion
                     <p>In the full MEI schema there are often multiple ways to encode something. MEI Basic 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
-                     applications adopt MEI and more use cases are identified. All MEI Basic files are 
-                     valid MEI "full" files, meaning MEI Basic files may be expanded and upgraded to MEI "full",
+                     applications adopt <abbr>MEI</abbr> and more use cases are identified. All <abbr>MEI</abbr> Basic files are 
+                     valid <abbr>MEI</abbr>"full" files, meaning <abbr>MEI</abbr> Basic files may be expanded and upgraded to <abbr>MEI</abbr>"full",
```
```suggestion
                     valid MEI <emph>full</emph> files, meaning MEI Basic files may be expanded and upgraded to MEI <emph>full</emph>,
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                   <div xml:id="pdfGuidelines" type="div4">
                      <head>Guidelines</head>
-                     <p>With MEI 5, we re-introduce a PDF version of the MEI Guidelines. With a total
+                     <p>With <abbr>MEI</abbr> 5, we re-introduce a PDF version of the <abbr>MEI</abbr> Guidelines. With a total
```
```suggestion
                     <p>With MEI 5, we re-introduce a PDF version of the MEI Guidelines. With a total
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
+                     <p>With <abbr>MEI</abbr> 5, we re-introduce a PDF version of the <abbr>MEI</abbr> Guidelines. With a total
                      of more than 5,700 pages, this PDF clearly is not intended to be printed, but may 
-                     serve as a single-file reference to the current release of MEI. The PDF is 
+                     serve as a single-file reference to the current release of <abbr>MEI</abbr>. The PDF is 
```
```suggestion
                     serve as a single-file reference to the current release of MEI. The PDF is 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                      <p>The Guidelines have also had several notable contributions, led in large part 
                      by our Interest Groups. These contributions have sought to make some chapters more
-                     clear and consistent, to help newcomers to MEI understand how MEI encoding may be
+                     clear and consistent, to help newcomers to <abbr>MEI</abbr> understand how <abbr>MEI</abbr> encoding may be
```
```suggestion
                     clear and consistent, to help newcomers to MEI understand how MEI encoding may be
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
-                     <p>In total, we have over 40 contributors actively involved in 
-                     the preparation of this release of MEI. Many of them are early-career researchers, 
-                     investing significant time and effort into the MEI Framework. Due to the open 
+                     <p>In total, we have over 40 contributors actively involved in "MEI"
```
```suggestion
                     <p>In total, we have over 40 contributors actively involved in 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                      other meetings, others may not be listed properly because of rather informal, 
                      but no less important, contributions. Without the joint effort of all those involved, 
-                     an undertaking like MEI would not be possible.</p>
+                     an undertaking like <abbr>MEI</abbr> would not be possible.</p>
```
```suggestion
                     an undertaking like MEI would not be possible.</p>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
                   <div xml:id="modelChanges" type="div4">
-                     <head>Model changes in MEI</head>
-                     <p>MEI 5.0 introduces five new elements: <gi scheme="MEI">plica</gi> and 
+                     <head>Model changes in <abbr>MEI</abbr></head>
```
```suggestion
                     <head>Model changes in MEI</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -214,73 +216,73 @@
-                     <head>Model changes in MEI</head>
-                     <p>MEI 5.0 introduces five new elements: <gi scheme="MEI">plica</gi> and 
+                     <head>Model changes in <abbr>MEI</abbr></head>
+                     <p><abbr>MEI</abbr> 5.0 introduces five new elements: <gi scheme="MEI">plica</gi> and 
```
```suggestion
                     <p>MEI 5.0 introduces five new elements: <gi scheme="MEI">plica</gi> and 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
-                     with <att>visible</att>="false". The text-rendition values of <val>letter-spacing</val> 
+                     <att>meter.form</att>=<val>invis</val> is updated to <att>meter.visible</att>=<val>false</val>, 
+                     and the same change applies to <att>form</att>=<val>invis</val> on meterSig, now replaced 
+                     with <att>visible</att>=<val>false</val>. The text-rendition values of <val>letter-spacing</val> 
```
```suggestion
                     with <att>visible</att> with <val>false</val>. The text-rendition values of <val>letter-spacing</val> 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
-                     <att>rend</att>="letter-spacing(0.25) line-height(120%)" will be now 
-                     <att>letterspacing</att>="0.25" <att>lineheight</att>="120%". Additionally, corrections 
+                     <att>rend</att>=<val>letter-spacing(0.25) line-height(120%)</val> will be now 
+                     <att>letterspacing</att>=<val>0.25</val> <att>lineheight</att>=<val>120%</val>. Additionally, corrections 
```
```suggestion
                     <att>letterspacing</att>=<val>0.25</val> <att>lineheight</att> with <val>120%</val>. Additionally, corrections 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
+                     group of changes affects the internal class structure of <abbr>MEI</abbr> only, where significant
                      effort went into improved consistency in naming things. While this set of changes 
-                     does not affect end users of MEI during validation of files, they may have 
+                     does not affect end users of <abbr>MEI</abbr> during validation of files, they may have 
```
```suggestion
                     does not affect end users of MEI during validation of files, they may have 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
                      We have also expanded our <ref target="https://github.com/music-encoding/music-encoding/blob/v5.0/CONTRIBUTING.md">Contribution Guidelines</ref> 
                      and other documentation files in the <ref target="https://github.com/music-encoding/music-encoding">music-encoding GitHub repository</ref>.</p>
-                     <p>The MEI documentation and guidelines are now expressed in TEI ODD again, 
+                     <p>The <abbr>MEI</abbr> documentation and guidelines are now expressed in <abbr>TEI</abbr> <abbr>ODD</abbr> again, 
```
```suggestion
                     <p>The MEI documentation and guidelines are now expressed in TEI ODD again, 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
                      The source code for both the Guidelines and the Specification is now jointly 
                      contained in the <ref target="https://github.com/music-encoding/music-encoding">music-encoding GitHub repository</ref>, 
-                     which simplifies validation across both parts of MEI. All assets – web 
+                     which simplifies validation across both parts of <abbr>MEI</abbr>. All assets – web 
```
```suggestion
                     which simplifies validation across both parts of MEI. All assets – web 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
                      documentation, PDF Guidelines, and schemata – are automatically generated from there. 
                      A multi-platform Docker image for running these processes locally is also provided
-                     to help new developers with getting started in contributing to MEI.
+                     to help new developers with getting started in contributing to <abbr>MEI</abbr>.
```
```suggestion
                     to help new developers with getting started in contributing to MEI.
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,49 +295,49 @@
                     <p>In addition to the main Music Encoding schema and Guidelines, we have also updated
                     our Sample Encodings and Encoding Tools repositories. Sample Encodings have been updated
-                    to MEI 5.0, and several problems with encodings from older releases have been fixed. In the
+                    to <abbr>MEI</abbr> 5.0, and several problems with encodings from older releases have been fixed. In the
```
```suggestion
                    to MEI 5.0, and several problems with encodings from older releases have been fixed. In the
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -349,9 +351,9 @@
             </div>
             <div xml:id="designPrinciples" type="div2">
-               <head>MEI Design Principles</head>
+               <head><abbr>MEI</abbr> Design Principles</head>
```
```suggestion
               <head>MEI Design Principles</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -406,23 +408,23 @@
                      indicating in the encoding the corresponding fields of other metadata
                      standards.</p>
-                  <p>To ensure broad international and multi-repertoire application of MEI,
+                  <p>To ensure broad international and multi-repertoire application of <abbr>MEI</abbr>,
```
```suggestion
                  <p>To ensure broad international and multi-repertoire application of MEI,
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
                      terms each year. Everyone registered to the <ref
                         target="https://music-encoding.org/community/community-contacts.html"
-                        >MEI-L</ref> mailing list is eligible to vote for the Board.</p>
+                        ><abbr>MEI-L</abbr></ref> mailing list is eligible to vote for the Board.</p>
```
```suggestion
                        >MEI-L</ref> mailing list is eligible to vote for the Board.</p>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
+                     maintenance and improvement of <abbr>MEI</abbr> itself. The Technical team will assist
                      Interest Groups and other interested community members in an advisory
-                     capacity on how to further develop MEI for both existing and new fields of
+                     capacity on how to further develop <abbr>MEI</abbr> for both existing and new fields of
```
```suggestion
                     capacity on how to further develop MEI for both existing and new fields of
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
             <div xml:id="basicConcepts" type="div2">
-               <head>Basic Concepts of MEI</head>
-               <p>This chapter is intended to explain basic concepts of MEI, like events vs.
+               <head>Basic Concepts of <abbr>MEI</abbr></head>
```
```suggestion
               <head>Basic Concepts of MEI</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
+                     <term>graphemic</term>, <term>acoustic</term> and <term>auditory</term>
+                     aspects of music (Babbitt, Milton: <title>The Use of Computers in Musicological Research</title>,
+                     in: <title>Perspectives of New Music</title> 3/2 (1965), p. 76).</p>
+                  <p>Various music encoding formats took up this distinction, most notably <abbr>SMDL</abbr>, the
```
```suggestion
                  <p>Various music encoding formats took up this distinction, most notably, the
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
                      10743). While the format itself was hardly ever used for its impractical
                      implementation details, parts of its design certainly influenced the
-                     development of other formats, including MEI. In a documentation draft (<ref
+                     development of other formats, including <abbr>MEI</abbr>. In a documentation draft (<ref
```
```suggestion
                     development of other formats, including MEI. In a documentation draft (<ref
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                        tempi, articulations, accents, etc.,” and it is the primary focus of SMDL.
-                        It can also be described as “the abstract information common to both the
-                        gestural and visual domains.” […]</item>
+                        tempi, articulations, accents, etc.,</q> and it is the primary focus of <abbr>SMDL</abbr>.
```
```suggestion
                        tempi, articulations, accents, etc.,</q> and it is the primary focus of <term>SMDL</term>.
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                     <hi rend="italic">conflating the logical and the visual domains</hi>. For example, MEI utilizes the
-                     <att>pname</att> (pitch name) attribute on notes to capture the <hi rend="italic">written</hi>
+                     the <abbr>MEI</abbr> schema, and quite frequently, two domains fall together in <abbr>MEI</abbr>. Very
+                     often, <emph><abbr>MEI</abbr> prioritizes the visual domain over the gestural domain</emph> by (partly)
```
```suggestion
                     often, <emph>MEI prioritizes the visual domain over the gestural domain</emph> by (partly)
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
                      to some degree, this does not mean that any given encoding has to provide
-                     visual information. MEI takes no assumption on what data is required: While an
-                     OMR project (<hi rend="italic">optical music recognition</hi>) may generate strictly visually
+                     visual information. <abbr>MEI</abbr> takes no assumption on what data is required: While an
```
```suggestion
                     visual information. MEI takes no assumption on what data is required: While an
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                     structure of MEI heavily relies on those different domains. When customizing
-                     MEI (see chapter <ptr target="#meiCustomization"/>), it is possible to turn off
-                     either visual or gestural domain encoding completely. That way, MEI allows to
+                     attributes carry a suffixed <att>.log</att> (logical), <att>.ges</att> (gestural), <att>.vis</att>
```
```suggestion
                     attributes carry a suffixed <code>.log</code> (logical), <code>.ges</code> (gestural), <code>.vis</code>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                     MEI (see chapter <ptr target="#meiCustomization"/>), it is possible to turn off
-                     either visual or gestural domain encoding completely. That way, MEI allows to
+                     attributes carry a suffixed <att>.log</att> (logical), <att>.ges</att> (gestural), <att>.vis</att>
+                     (visual), or <att>.anl</att> (analytical) in their name. In addition, the internal
```
```suggestion
                     (visual), or <code>.anl</code> (analytical) in their name. In addition, the internal
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                     either visual or gestural domain encoding completely. That way, MEI allows to
+                     attributes carry a suffixed <att>.log</att> (logical), <att>.ges</att> (gestural), <att>.vis</att>
+                     (visual), or <att>.anl</att> (analytical) in their name. In addition, the internal
+                     structure of <abbr>MEI</abbr> heavily relies on those different domains. When customizing
```
```suggestion
                     structure of MEI heavily relies on those different domains. When customizing
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
+                     attributes carry a suffixed <att>.log</att> (logical), <att>.ges</att> (gestural), <att>.vis</att>
+                     (visual), or <att>.anl</att> (analytical) in their name. In addition, the internal
+                     structure of <abbr>MEI</abbr> heavily relies on those different domains. When customizing
+                     <abbr>MEI</abbr>(see chapter <ptr target="#meiCustomization"/>), it is possible to turn off
```
```suggestion
                     MEI(see chapter <ptr target="#meiCustomization"/>), it is possible to turn off
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
+                     (visual), or <att>.anl</att> (analytical) in their name. In addition, the internal
+                     structure of <abbr>MEI</abbr> heavily relies on those different domains. When customizing
+                     <abbr>MEI</abbr>(see chapter <ptr target="#meiCustomization"/>), it is possible to turn off
+                     either visual or gestural domain encoding completely. That way, <abbr>MEI</abbr> allows to
```
```suggestion
                     either visual or gestural domain encoding completely. That way, MEI allows to
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -442,145 +444,142 @@
-                        >measure</gi> they apply to, following all <gi scheme="MEI">staff</gi>
+                     Simply put, <term>Events</term> describe <emph>what</emph> needs to be performed, and <term>ControlEvents</term>
+                     indicate <emph>how</emph> it needs to be performed. In (<ptr
+                        target="#cmn"/>-based) <abbr>MEI</abbr>, <term>Events</term> are nested inside
```
```suggestion
                        target="#cmn"/>-based) MEI, <term>Events</term> are nested inside
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -661,15 +660,14 @@
-                     >Events</hi>.</p>
-                  <p>The details on how timestamps are calculated and used in MEI are given in <ptr
+                     connect <term>ControlEvents</term> with <term>Events</term>.</p>
+                  <p>The details on how timestamps are calculated and used in <abbr>MEI</abbr> are given in <ptr
```
```suggestion
                  <p>The details on how timestamps are calculated and used in MEI are given in <ptr
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -661,15 +660,14 @@
-                  <head>Timestamps in MEI</head>
-                  <p>In MEI, timestamps are treated in a slightly simplified way: they have no
-                     notion of <hi rend="italic">beat</hi>. Instead, timestamps rely solely on the
+                  <head>Timestamps in <abbr>MEI</abbr></head>
```
```suggestion
                  <head>Timestamps in MEI</head>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
-                  <p>MEI is an encoding framework, not a data format. This means that MEI provides
-                     recommendations for encoding music documents, but it depends on the encoder's
+                  <head><abbr>MEI</abbr> Profiles</head>
+                  <p><abbr>MEI</abbr> is an encoding framework, not a data format. This means that <abbr>MEI</abbr> provides
```
```suggestion
                  <p>MEI is an encoding framework, not a data format. This means that MEI provides
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
                      types and music repertoires, but it is rarely advisable to use them all side by
                      side in one encoding.</p>
-                  <p>In order to use MEI, it is advised to use a restricted version of the schema,
+                  <p>In order to use <abbr>MEI</abbr>, it is advised to use a restricted version of the schema,
```
```suggestion
                  <p>In order to use MEI, it is advised to use a restricted version of the schema,
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
                      which will make it easier both for an encoder and a reader of the encoded
-                     files. MEI provides a number of pre-defined <hi rend="italic">profiles</hi>, which focus on specific
-                     uses of MEI while still maintaining a great level of flexibility. For projects
+                     files. <abbr>MEI</abbr> provides a number of pre-defined <term>profiles</term>, which focus on specific
```
```suggestion
                     files. MEI provides a number of pre-defined <term>profiles</term>, which focus on specific
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
+                     create a more specific customized version of <abbr>MEI</abbr>(see chapter <ptr
                         target="#meiCustomization"/>). The following customizations are provided
-                     with every release of MEI:</p>
+                     with every release of <abbr>MEI</abbr>:</p>
```
```suggestion
                     with every release of MEI:</p>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
-                     <item>For documents written in <hi rend="italic">Mensural Notation</hi> (both black and
-                        white), MEI offers the <hi rend="italic">mei-Mensural</hi> customization. The specific rules for
+                     <item>For documents written in <term>Mensural Notation</term> (both black and
+                        white), <abbr>MEI</abbr> offers the <term>mei-Mensural</term> customization. The specific rules for
```
```suggestion
                        white), MEI offers the <term>mei-Mensural</term> customization. The specific rules for
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
-                        developing software and converters for MEI. The <hi rend="italic">mei-Basic</hi> profile is a subset
-                        of MEI which restricts it to one way of encoding for every feature of music
-                        notation. It covers <hi rend="italic">Common Western Music Notation</hi> only, and excludes all
+                        core of <abbr>MEI</abbr> and often required for research projects, it is an obstacle when
```
```suggestion
                        core of MEI and often required for research projects, it is an obstacle when
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
-                        notation. It covers <hi rend="italic">Common Western Music Notation</hi> only, and excludes all
+                        core of <abbr>MEI</abbr> and often required for research projects, it is an obstacle when
+                        developing software and converters for <abbr>MEI</abbr>. The <term>mei-Basic</term> profile is a subset
+                        of <abbr>MEI</abbr> which restricts it to one way of encoding for every feature of music
```
```suggestion
                        of MEI which restricts it to one way of encoding for every feature of music
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
+                        encoding formats like MusicXML or <abbr>MNX</abbr>. The purpose of <term>mei-Basic</term> is to serve
                         as common ground for data interchange, both between projects using different
-                        profiles of MEI, and other encoding schemes.</item>
+                        profiles of <abbr>MEI</abbr>, and other encoding schemes.</item>
```
```suggestion
                        profiles of MEI, and other encoding schemes.</item>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,76 +699,75 @@
                         which in turn contain staves. These staves have a different meaning here, and
-                        are modeled differently in MEI. <hi rend="italic">mei-all</hi> mixes those models and thus invites
-                        encoding errors. In general, you should almost never use <hi rend="italic">mei-all</hi> except for
+                        are modeled differently in <abbr>MEI</abbr>. <term>mei-all</term> mixes those models and thus invites
```
```suggestion
                        are modeled differently in MEI. <term>mei-all</term> mixes those models and thus invites
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -782,33 +779,33 @@
                      help to ensure consistency and data quality throughout a project’s files. A
-                     customization typically provides a subset of MEI’s encoding models (typically
-                     starting from one of the official <hi rend="italic">profiles</hi> mentioned in chapter <ptr
+                     customization typically provides a subset of <abbr>MEI</abbr>’s encoding models (typically
```
```suggestion
                     customization typically provides a subset of MEI’s encoding models (typically
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -782,33 +779,33 @@
-                     XML-based markup language developed and maintained by the TEI. TEI's
-                     documentation for ODD can be found in the TEI Guidelines chapter 22: <ref
+                  <p><abbr>MEI</abbr> is implemented in <abbr>ODD</abbr>. <abbr>ODD</abbr>, or <term>One Document Does-it-all</term>, is another
+                     <abbr>XML</abbr>-based markup language developed and maintained by the <abbr>TEI</abbr>. <abbr>TEI</abbr>’s
```
```suggestion
                     XML-based markup language developed and maintained by the TEI. TEI’s
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -782,33 +779,33 @@
-                  <p>At this point, there is no specific documentation on how to customize MEI with
-                     ODD beyond the generic TEI documentation. However, the provided <ptr
-                        target="#meiProfiles"/> are based on ODD customizations, and may serve as
+                        >Getting Started with P5 <abbr>ODD</abbr>s</ref></title> document.</p>
```
```suggestion
                        >Getting Started with P5 ODDs</ref></title> document.</p>
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -782,33 +779,33 @@
-                     ODD beyond the generic TEI documentation. However, the provided <ptr
-                        target="#meiProfiles"/> are based on ODD customizations, and may serve as
+                        >Getting Started with P5 <abbr>ODD</abbr>s</ref></title> document.</p>
+                  <p>At this point, there is no specific documentation on how to customize <abbr>MEI</abbr> with
```
```suggestion
                  <p>At this point, there is no specific documentation on how to customize MEI with
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -782,33 +779,33 @@
-                        target="#meiProfiles"/> are based on ODD customizations, and may serve as
+                        >Getting Started with P5 <abbr>ODD</abbr>s</ref></title> document.</p>
+                  <p>At this point, there is no specific documentation on how to customize <abbr>MEI</abbr> with
+                     <abbr>ODD</abbr> beyond the generic <abbr>TEI</abbr> documentation. However, the provided <ptr
```
```suggestion
                     <term>ODD</term> beyond the generic <abbr>TEI</abbr> documentation. However, the provided <ptr
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -819,24 +816,24 @@
                         target="https://github.com/BeethovensWerkstatt/module2/tree/dev/data/odd"
                         >Beethovens Werkstatt</ref>.</p>
-                  <p>MEI provides a web service at <ref target="http://custom.music-encoding.org/"
+                  <p><abbr>MEI</abbr> provides a web service at <ref target="http://custom.music-encoding.org/"
```
```suggestion
                  <p>MEI provides a web service at <ref target="http://custom.music-encoding.org/"
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -819,24 +816,24 @@
+                  <p><abbr>MEI</abbr> provides a web service at <ref target="http://custom.music-encoding.org/"
                         >http://custom.music-encoding.org</ref> which allows to
-                     compile such customizations against the MEI sources in order to generate
+                     compile such customizations against the <abbr>MEI</abbr> sources in order to generate
```
```suggestion
                     compile such customizations against the MEI sources in order to generate
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -819,24 +816,24 @@
+                     compile such customizations against the <abbr>MEI</abbr> sources in order to generate
                      RelaxNG schemata, which can be used for validation. More documentation on
-                     customizing MEI will be provided as time permits; until then, it is recommended
+                     customizing <abbr>MEI</abbr> will be provided as time permits; until then, it is recommended
```
```suggestion
                     customizing MEI will be provided as time permits; until then, it is recommended
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -819,24 +816,24 @@
             </div>
             <div xml:id="samplesTools" type="div2">
-               <head>Sample Encodings and Tools for MEI</head>
+               <head>Sample Encodings and Tools for <abbr>MEI</abbr></head>
```
```suggestion
               <head>Sample Encodings and Tools for MEI</head>
```

krHERO commented on source/docs/02-shared.xml:
```diff
@@ -324,7 +324,7 @@
                      </specList>
                   </p>
-                  <p>"Forme work" is the name for running elements (page headers and footers). Both <gi scheme="MEI">pgHead</gi> and <gi scheme="MEI">pgFoot</gi> have a <att>func</att> attribute that allows encoders to specify to which page(s) the forme work element applies. This includes alternating patterns.</p>
+                  <p><hi rend="italic">Forme work</hi> is the name for running elements (page headers and footers). Both <gi scheme="MEI">pgHead</gi> and <gi scheme="MEI">pgFoot</gi> have a <att>func</att> attribute that allows encoders to specify to which page(s) the forme work element applies. This includes alternating patterns.</p>
```
```suggestion
                  <p><emph>Forme work</emph> is the name for running elements (page headers and footers). Both <gi scheme="MEI">pgHead</gi> and <gi scheme="MEI">pgFoot</gi> have a <att>func</att> attribute that allows encoders to specify to which page(s) the forme work element applies. This includes alternating patterns.</p>
```

krHERO commented on source/docs/02-shared.xml:
```diff
@@ -476,10 +476,10 @@
-                     <p>‘Space’ can be thought of as another kind of event. In fact, some refer to this concept as an ‘invisible rest’.</p>
-                     <p>While ‘space’ is meaningful, ‘padding’ is non-essential whitespace that is used to shift the position of the events which follow.</p>
+                     <p><hi rend="italic">Space</hi> can be thought of as another kind of event. In fact, some refer to this concept as an <hi rend="italic">invisible rest</hi>.</p>
+                     <p>While <hi rend="italic">space</hi> is meaningful, <hi rend="italic">padding</hi> is non-essential whitespace that is used to shift the position of the events which follow.</p>
```
```suggestion
                     <p>While <emph>space</emph> is meaningful, <emph>padding</emph> is non-essential whitespace that is used to shift the position of the events which follow.</p>
```

krHERO commented on source/docs/02-shared.xml:
```diff
@@ -499,7 +499,7 @@
                         </specList>
                      </p>
-                     <p>Examples of directives include text strings such as 'affettuoso', fingering numbers, or music symbols such as segno and coda symbols or fermatas over a bar line. Directives can be control elements. That is, they can linked via their attributes to other events. The starting point of the directive may be indicated by either a tstamp, tstamp.ges, tstamp.real or startid attrib ...
+                     <p>Examples of directives include text strings such as <hi rend="italic">affettuoso</hi>, fingering numbers, or music symbols such as segno and coda symbols or fermatas over a bar line. Directives can be control elements. That is, they can linked via their attributes to other events. The starting point of the directive may be indicated by either a tstamp, tstamp.ges, tstamp.r ...
```
```suggestion
                     <p>Examples of directives include text strings such as <emph>affettuoso</emph>, fingering numbers, or music symbols such as segno and coda symbols or fermatas over a bar line. Directives can be control elements. That is, they can linked via their attributes to other events. The starting point of the directive may be indicated by either a tstamp, tstamp.ges, tstamp.real or startid attribute, while the ending point may be recorded by either a tstamp2, dur, dur.ges or endid attribute. It is a semantic error not to specify a starting point attribute.</p>
```

krHERO commented on source/docs/03-metadata.xml:
```diff
@@ -333,10 +333,10 @@
                      <p>The <gi scheme="MEI">extent</gi> element describes the approximate size of a text as stored on some carrier medium, whether digital or non-digital, specified in any convenient units.</p>
                      <p>For printed books, information about the carrier, such as the kind of medium used and its size, are of great importance in cataloging procedures. The print-oriented rules for bibliographic description of an item’s medium and extent need some re-interpretation when applied to electronic media. An electronic file exists as a distinct entity quite independently of its carrier ...
-                     <p>This is particularly true of information about file-type although library-oriented rules for cataloging often distinguish two types of computer file: ‘data’ and ‘programs’. This distinction is quite difficult to draw in some cases, for example, hypermedia or texts with built-in search and retrieval software.</p>
+                     <p>This is particularly true of information about file-type although library-oriented rules for cataloging often distinguish two types of computer file: <hi rend="italic">data</hi> and <hi rend="italic">programs</hi>. This distinction is quite difficult to draw in some cases, for example, hypermedia or texts with built-in search and retrieval software.</p>
```
```suggestion
                     <p>This is particularly true of information about file-type although library-oriented rules for cataloging often distinguish two types of computer file: <emph>data</emph> and <emph>programs</emph>. This distinction is quite difficult to draw in some cases, for example, hypermedia or texts with built-in search and retrieval software.</p>
```

musicEnfanthen replied:
emph?

krHERO commented on source/docs/03-metadata.xml:
```diff
@@ -712,13 +712,13 @@
                <list type="gloss">
                   <label>Work</label>
-                  <item>FRBR defines a work as a <quote>distinct intellectual or artistic creation</quote>, an abstract entity because there is no single material object one can point to as the work.</item>
+                  <item><abbr>FRBR</abbr> defines a work as a <quote>distinct intellectual or artistic creation</quote>, an abstract entity because there is no single material object one can point to as the work.</item>
```
```suggestion
                  <item>FRBR defines a work as a <quote>distinct intellectual or artistic creation</quote>, an abstract entity because there is no single material object one can point to as the work.</item>
```

krHERO commented on source/docs/04-cmn.xml:
```diff
@@ -106,7 +106,7 @@
                <div xml:id="cmnMeasures" type="div3">
                   <head>The Role of the Measure Element</head>
-                  <p>Arguably, the most important element of the CMN module is the <gi scheme="MEI">measure</gi> element. It is used as a structural unit inside <gi scheme="MEI">section</gi> elements and acts as a container for ‘events’ from the <ident type="class">model.eventLike</ident> class, such as notes, chords and rests as well as ‘control events’ from the <ident type="class">model.control ...
+                  <p>Arguably, the most important element of the CMN module is the <gi scheme="MEI">measure</gi> element. It is used as a structural unit inside <gi scheme="MEI">section</gi> elements and acts as a container for <hi rend="italic">events</hi> from the <ident type="class">model.eventLike</ident> class, such as notes, chords and rests as well as <hi rend="italic">control events</hi>  ...
```
```suggestion
                  <p>Arguably, the most important element of the CMN module is the <gi scheme="MEI">measure</gi> element. It is used as a structural unit inside <gi scheme="MEI">section</gi> elements and acts as a container for <emph>events</emph> from the <ident type="class">model.eventLike</ident> class, such as notes, chords and rests as well as <emph>control events</emph> from the <ident type="class">model.controlEventLike</ident> class, such as slurs and indications of dynamics.</p>
```

krHERO commented on source/docs/04-cmn.xml:
```diff
@@ -215,7 +215,7 @@
                   <p>With the exception of <gi scheme="MEI">label</gi>, these elements may also occur within the flow of musical events captured in a <gi scheme="MEI">layer</gi>, since they are members of <ident type="class">model.eventLike</ident>. In the layer context they function as milestones and affect all following content assigned to the layer (even in subsequent measures) until their inf ...
                   <p>Such flexibility as this may require close inspection of an encoding to retrieve the correct definitions for a given staff. As a general rule, the closest preceding and most specific element provides this information: For example, a <gi scheme="MEI">keySig</gi> in the preceding measure is more relevant than a <gi scheme="MEI">staffDef</gi> at the beginning of the section, whi ...
-                  <p>Every <gi scheme="MEI">staffDef</gi> may contain a number of <gi scheme="MEI">layerDef</gi> elements, which may be used to establish default values for the distinct layers sharing one staff. MEI does not use the term ‘voice’ to describe these ‘musical threads’ because that term implies continuity across measure boundaries. Given the sometimes arbitrary relationships between t ...
+                  <p>Every <gi scheme="MEI">staffDef</gi> may contain a number of <gi scheme="MEI">layerDef</gi> elements, which may be used to establish default values for the distinct layers sharing one staff. MEI does not use the term <hi rend="italic">voice</hi> to describe these <q>musical threads</q> because that term implies continuity across measure boundaries. Given the sometimes arbitra ...
```
```suggestion
                  <p>Every <gi scheme="MEI">staffDef</gi> may contain a number of <gi scheme="MEI">layerDef</gi> elements, which may be used to establish default values for the distinct layers sharing one staff. MEI does not use the term <hi rend="italic">voice</hi> to describe these <q>musical threads</q> because that term implies continuity across measure boundaries. Given the sometimes arbitrary relationships between these threads from measure to measure as well as across staves, MEI uses the more neutral term <emph>layer</emph></p>
```

krHERO commented on source/docs/04-cmn.xml:
```diff
@@ -680,7 +680,7 @@
                      </figure>
                   </p>
-                  <p>Obviously, to be complete the slur in the above example needs to be ‘attached’ to the notes somehow. The ‘vertical assignment’ can be indicated for the example above using the <att>staff</att> and <att>layer</att> attributes like so:</p>
+                  <p>Obviously, to be complete the slur in the above example needs to be <q>attached</q> to the notes somehow. The <hi rend="italic">vertical assignment</hi> can be indicated for the example above using the <att>staff</att> and <att>layer</att> attributes like so:</p>
```
```suggestion
                  <p>Obviously, to be complete the slur in the above example needs to be <q>attached</q> to the notes somehow. The <q>vertical assignment</q> can be indicated for the example above using the <att>staff</att> and <att>layer</att> attributes like so:</p>
```

krHERO commented on source/docs/04-cmn.xml:
```diff
@@ -900,7 +900,7 @@
                      </p>
                      <p>A third, and somewhat counter-intuitive, value for <att>order</att>, <val>nonarp</val>, indicates that no arpeggio shall be performed. Normally rendered as a bracket instead of a wavy line, this form of arpeggio is used to indicate a non-arpeggiated chord intervening in a sequence of arpeggiated ones. This is common in music for the harp, where arpeggiation is the usual me ...
-                     <p>Whereas an arpeggio ‘staggers’ the onset times of the notes of a chord, a glissando denotes a situation where the pitch ‘slides’ from one note to another. It makes no difference whether this slide produces distinct intermediate pitches (as on the piano) or not (as on the trombone), though the latter is sometimes referred to as portamento. The visual appearance of a glissan ...
+                     <p>Whereas an arpeggio <hi rend="italic">staggers</hi> the onset times of the notes of a chord, a glissando denotes a situation where the pitch ‘slides’ from one note to another. It makes no difference whether this slide produces distinct intermediate pitches (as on the piano) or not (as on the trombone), though the latter is sometimes referred to as portamento. The visual ap ...
```
```suggestion
                     <p>Whereas an arpeggio <q>staggers</q> the onset times of the notes of a chord, a glissando denotes a situation where the pitch <q>slides</q> from one note to another. It makes no difference whether this slide produces distinct intermediate pitches (as on the piano) or not (as on the trombone), though the latter is sometimes referred to as portamento. The visual appearance of a glissando, which MEI encodes as <gi scheme="MEI">gliss</gi>, is normally a line connecting two notes in the glissando.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -99,7 +99,7 @@
             <div xml:id="neumesRepertoire" type="div2">
                <head>Overview of the Neumes Module</head>
-               <p>The MEI Neumes Module represents the community’s attempt to create a standardized set of rules that encapsulate in a logical, systematic, and unequivocal way the musical information represented and conveyed by Western European neumatic notations (beginning with the late ninth century and continuing to the printed books of the twentieth). Most neume notation is used to set music  ...
+               <p>The MEI Neumes Module represents the community’s attempt to create a standardized set of rules that encapsulate in a logical, systematic, and unequivocal way the musical information represented and conveyed by Western European neumatic notations (beginning with the late ninth century and continuing to the printed books of the twentieth). Most neume notation is used to set music  ...
```
```suggestion
               <p>The MEI Neumes Module represents the community’s attempt to create a standardized set of rules that encapsulate in a logical, systematic, and unequivocal way the musical information represented and conveyed by Western European neumatic notations (beginning with the late ninth century and continuing to the printed books of the twentieth). Most neume notation is used to set music to an existing text. The syllable is the fundamental unit of structure, with the neumes themselves serving as a means of <q>sonifying</q> the text. A syllable may be expressed via one or more neumes, with the particular neume shape chosen depending on the pitch contour that is being employed and the desired interpretation.</p>
```

krHERO commented on source/docs/09-textencoding.xml:
```diff
@@ -149,26 +149,26 @@
                   <div xml:id="textFront" type="div4">
                      <head>Front Matter</head>
-                     <p>By ‘front matter’ these Guidelines mean distinct sections of a text (usually, but not necessarily, a printed one), prefixed to it by way of introduction or identification as a part of its production. The ‘front matter’ is encoded with the <gi scheme="MEI">front</gi> element.</p>
+                     <p>By <hi rend="italic">front matter</hi> these Guidelines mean distinct sections of a text (usually, but not necessarily, a printed one), prefixed to it by way of introduction or identification as a part of its production. The <hi rend="italic">front matter</hi> is encoded with the <gi scheme="MEI">front</gi> element.</p>
```
```suggestion
                     <p>By <q>front matter</q> these Guidelines mean distinct sections of a text (usually, but not necessarily, a printed one), prefixed to it by way of introduction or identification as a part of its production. The front matter is encoded with the <gi scheme="MEI">front</gi> element.</p>
```

krHERO commented on source/docs/09-textencoding.xml:
```diff
@@ -271,7 +271,7 @@
                <div xml:id="sharedTextRendition" type="div3">
                   <head>Text Rendition</head>
-                  <p>Sometimes, it is desirable to capture the typographical qualities of a word or phrase without assigning it a special meaning. For this purpose, MEI offers the <gi scheme="MEI">rend</gi> element, similar to TEI’s <hi rend="italic">hi</hi> element. Using CSS-like values, its <att>rend</att> attribute can be used to specify many typographic features, such as font style, font var ...
+                  <p>Sometimes, it is desirable to capture the typographical qualities of a word or phrase without assigning it a special meaning. For this purpose, MEI offers the <gi scheme="MEI">rend</gi> element, similar to <abbr>TEI</abbr>’s <hi rend="italic">hi</hi> element. Using CSS-like values, its <att>rend</att> attribute can be used to specify many typographic features, such as font st ...
```
```suggestion
                  <p>Sometimes, it is desirable to capture the typographical qualities of a word or phrase without assigning it a special meaning. For this purpose, MEI offers the <gi scheme="MEI">rend</gi> element, similar to TEI’s <hi rend="italic">hi</hi> element. Using CSS-like values, its <att>rend</att> attribute can be used to specify many typographic features, such as font style, font variants, and relative font size and weight. In addition, text decoration, direction, and enclosing <q>boxes</q> may be captured. While <att>rend</att> is used to record relative font size and weight, absolute values for these qualities (measured in printer’s points) should be specified using the <att>fontsize</att> and <att>fontweight</att> attributes. In addition to commonly found typographical qualities, MEI provides the <att>altrend</att> attribute for the capture of additional, user-defined rendition information.</p>
```

krHERO reviewed (commented):
Hi @ArturJD96 , thanks for all the work you already put into this PR!

 I tried to consult all the changes and reviews and comments to try to help to get this PR merge-able. While doing so and reading the review comments/suggestions, I had the feeling, the issue and tasks that was asked for, could have been more precisely, maybe with the help of some examples. So I recommend in general to think about creating issue (and PR) templates as [Verovio](https://github.com/rism-digital/verovio/tree/develop/.github/ISSUE_TEMPLATE) and Edirom ([issue](https://github.com/Edirom/Edirom-Online/tree/develop/.github/ISSUE_TEMPLATE) and [PR](https://github.com/Edirom/Edirom-Online/blob/develop/.github/pull_request_template.md)) already does, to make it easier to get started, especially when issues have the Label `Type: Good first issue`. 

@ArturJD96, I would appreciate it a lot, if you could have a look at my review and code suggestions.

krHERO reviewed (commented):
Hi @ArturJD96 , thanks for all the work you already put into this PR!

 I tried to consult all the changes and reviews and comments to try to help to get this PR merge-able. While doing so and reading the review comments/suggestions, I had the feeling, the issue and tasks that was asked for, could have been more precisely, maybe with the help of some examples. So I recommend in general to think about creating issue (and PR) templates as [Verovio](https://github.com/rism-digital/verovio/tree/develop/.github/ISSUE_TEMPLATE) and Edirom ([issue](https://github.com/Edirom/Edirom-Online/tree/develop/.github/ISSUE_TEMPLATE) and [PR](https://github.com/Edirom/Edirom-Online/blob/develop/.github/pull_request_template.md)) already does, to make it easier to get started, especially when issues have the Label `Type: Good first issue`. 

@ArturJD96, I would appreciate it a lot, if you could have a look at my review and code suggestions.

bwbohl committed:
Apply suggestion from @krHERO

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply first 50 suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply another 50 suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply another 45 suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

bwbohl committed:
Apply suggestions from code review for encoding `<att>` and `<val>`combinations

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

bwbohl commented on source/docs/01-introduction.xml:
```diff
@@ -70,19 +70,19 @@
+                <p>This is a derivative work based on earlier versions of the schema copyright (c)
+                    2001-2006 Perry Roland and the Rector and Visitors of the <institution>University of Virginia</institution>;
+                    licensed under the Educational Community License version 1.0.</p>
+               <p>CONTACT: <email>contact@music-encoding.org</email> </p>
```
probably header changes should be reverted locally

musicEnfanthen committed:
Apply suggestion from @bwbohl

Co-authored-by: Benjamin W. Bohl <b.w.bohl@gmail.com>

musicEnfanthen commented on source/docs/01-introduction.xml:
```diff
@@ -716,46 +711,46 @@
                      <label>mei-Mensural</label>
-                     <item>For documents written in <hi rend="italic">Mensural Notation</hi> (both black and
-                        white), MEI offers the <hi rend="italic">mei-Mensural</hi> customization. The specific rules for
+                     <item>For documents written in <term>Mensural Notation</term> (both black and
```
emph

musicEnfanthen commented on source/docs/04-cmn.xml:
```suggestion
                  <p>Each <gi scheme="MEI">staff</gi> may hold a number of <gi scheme="MEI">layer</gi> elements to reflect multiple <emph>voices</emph>. Just as with <gi scheme="MEI">staff</gi>, the order of the <gi scheme="MEI">layer</gi> elements in the file does not have to reflect their original order in the document, so they also possess <att>n</att> and <att>def</att> attributes for association with the appropriate layer definition.</p>
```

musicEnfanthen committed:
Apply suggestion from @krHERO

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestion from @krHERO

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,12 +293,12 @@
-                     <att>meter.form</att>="invis" is updated to <att>meter.visible</att>="false", 
-                     and the same change applies to <att>form</att>="invis" on meterSig, now replaced 
-                     with <att>visible</att>="false". The text-rendition values of <val>letter-spacing</val> 
+                     <att>meter.form</att> with <val>invis</val> is updated to <att>meter.visible</att> with <val>false</val>, 
```
```suggestion
                     <att>meter.form</att>=<val>invis</val> is updated to <att>meter.visible</att>=<val>false</val>, 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,12 +293,12 @@
-                     with <att>visible</att>="false". The text-rendition values of <val>letter-spacing</val> 
+                     <att>meter.form</att> with <val>invis</val> is updated to <att>meter.visible</att> with <val>false</val>, 
+                     and the same change applies to <att>form</att> with <val>invis</val> on meterSig, now replaced 
+                     with <att>visible</att> with <val>false</val>. The text-rendition values of <val>letter-spacing</val> 
```
```suggestion
                     with <att>visible</att>=<val>false</val>. The text-rendition values of <val>letter-spacing</val> 
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -293,12 +293,12 @@
-                     <att>rend</att>="letter-spacing(0.25) line-height(120%)" will be now 
-                     <att>letterspacing</att>="0.25" <att>lineheight</att>="120%". Additionally, corrections 
+                     <att>rend</att> with <val>letter-spacing(0.25) line-height(120%)</val> will be now 
+                     <att>letterspacing</att>=<val>0.25</val> <att>lineheight</att> with <val>120%</val>. Additionally, corrections 
```
```suggestion
                     <att>letterspacing</att>=<val>0.25</val> <att>lineheight</att>=<val>120%</val>. Additionally, corrections 
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -461,45 +461,44 @@
-                        >http://xml.coverpages.org/smdl10743-pdf.gz, p.5</ref>), SMDL identifies
-                     four different <hi rend="italic">musical domains</hi>:</p>
+                        >http://xml.coverpages.org/smdl10743-pdf.gz, p.5</ref>), <abbr>SMDL</abbr> identifies
+                     four different <term>musical domains</term>:</p>
```
```suggestion
                     four different <emph>musical domains</emph>:</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>
Co-authored-by: Benjamin W. Bohl <b.w.bohl@gmail.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
                      are other examples for such a separation of concerns with regard to music. In
-                     Greg’s Copy-Text Theory (W.W. Greg: <hi rend="italic">The Rationale of
-                        Copy-Text</hi>, 1950), a distinction between primary and secondary text is
+                     Greg’s <term>Copy-Text Theory</term> (W.W. Greg: <title>The Rationale of
```
```suggestion
                     Greg’s Copy-Text Theory (W.W. Greg: <hi rend="italic">The Rationale of
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
                      made; similar attempts have been made for music specifically.</p>
-                  <p>In MEI, elements describing the basic musical text are referred to as <hi
-                        rend="italic">Events</hi>. They are the building blocks for the stream of
+                  <p>In MEI, elements describing the basic musical text are referred to as <term>Events</term>.
```
```suggestion
                  <p>In MEI, elements describing the basic musical text are referred to as <emph>Events</emph>.
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
-                        >rest</gi>s, and <gi scheme="MEI">chord</gi>s. In contrast, <hi
-                        rend="italic">ControlEvents</hi> make no independent contribution to that
+                        >rest</gi>s, and <gi scheme="MEI">chord</gi>s. In contrast,
+                        <term>ControlEvents</term> make no independent contribution to that
```
```suggestion
                        <emph>ControlEvents</emph> make no independent contribution to that
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
                      flow of music. Instead, they provide additional information about the encoded
-                        <hi rend="italic">Events</hi>, they <hi rend="italic">control</hi> their
-                     performance. Examples for such <hi rend="italic">ControlEvents</hi> are <gi
+                        <term>Events</term>, they <emph>control</emph> their
```
```suggestion
                        <emph>Events</emph>, they <emph>control</emph> their
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
-                     a <gi scheme="MEI">layer</gi> element, while <hi rend="italic"
-                        >ControlEvents</hi> are direct children of the first <gi scheme="MEI"
-                        >measure</gi> they apply to, following all <gi scheme="MEI">staff</gi>
+                     Simply put, <term>Events</term> describe <emph>what</emph> needs to be performed, and <term>ControlEvents</term>
```
```suggestion
                     Simply put, <emph>Events</emph> describe <emph>what</emph> needs to be performed, and <emph>ControlEvents</emph>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -604,16 +601,16 @@
-                     XML document changes for whatever reason, they will keep that connection. This
-                     means that the <hi rend="italic">semantic position</hi> to which they are bound
+                  <p>A <term>ControlEvent</term> encoded like above will be strictly tied
+                     to the referenced <term>Events</term> – if their position inside the
```
```suggestion
                     to the referenced <emph>Events</emph> – if their position inside the
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -525,62 +524,60 @@
-                     established by using <hi rend="italic">pointers</hi>. In this case, all events
+                     benefits and drawbacks. A technically very stable connection between
+                     <term>ControlEvents</term> and <term>Events</term> can be
+                     established by using <term>pointers</term>. In this case, all events
```
```suggestion
                     established by using <emph>pointers</emph>. In this case, all events
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -635,19 +632,19 @@
-                     different pitch and / or duration. Actually, it is not required that an <hi
-                        rend="italic">Event</hi> can be found at the position indicated by a
+                     different pitch and / or duration. Actually, it is not required that
+                     an <term>Event</term> can be found at the position indicated by a
```
```suggestion
                     an <emph>Event</emph> can be found at the position indicated by a
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -635,19 +632,19 @@
                      between two events, or dynam markings spread across otherwise empty
                      measures.</p>
-                  <p>If the ending of a <hi rend="italic">ControlEvent</hi> shall be given by
+                  <p>If the ending of a <term>ControlEvent</term> shall be given by
```
```suggestion
                  <p>If the ending of a <emph>ControlEvent</emph> shall be given by
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -701,8 +697,7 @@
                   </p>
-                  <p>Sometimes, timestamps are used to indicate positions where no music <hi
-                        rend="italic">Events</hi> are located (see <ptr
+                  <p>Sometimes, timestamps are used to indicate positions where no music <term>Events</term> are located (see <ptr
```
```suggestion
                  <p>Sometimes, timestamps are used to indicate positions where no music <emph>Events</emph> are located (see <ptr
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -716,46 +711,46 @@
                      uses of MEI while still maintaining a great level of flexibility. For projects
                      that need even better control over their data, it is highly recommended to
-                     create a more specific customized version of MEI (see chapter <ptr
+                     create a more specific customized version of MEI(see chapter <ptr
```
```suggestion
                     create a more specific customized version of MEI (see chapter <ptr
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -716,46 +711,46 @@
-                        encoding with MEI. The <hi rend="italic">mei-CMN</hi> customization targets at documents that use
-                        <hi rend="italic">Common Western Music Notation</hi>. The specific rules for that notation are
+                        encoding with MEI. The <term>mei-CMN</term> customization targets at documents that use
+                        <term>Common Western Music Notation</term>. The specific rules for that notation are
```
```suggestion
                        <emph>Common Western Music Notation</emph>. The specific rules for that notation are
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -716,46 +711,46 @@
                         The specific rules for that notation are specified in chapter <ptr
                            target="#neumes"/>, even though other chapters of these Guidelines apply as
-                        well. Please note that the <hi rend="italic">mei-Neumes</hi> profile has undergone significant
+                        well. Please note that the <term>mei-Neumes</term> profile has undergone significant
```
```suggestion
                        well. Please note that the <emph>mei-Neumes</emph> profile has undergone significant
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -764,11 +759,11 @@
                         testing purposes.</item>
                      <label>mei-all_anyStart </label>
-                     <item>This profile includes all of <hi rend="italic">mei-all</hi>, but extends it even
+                     <item>This profile includes all of <term>mei-all</term>, but extends it even
```
```suggestion
                     <item>This profile includes all of <emph>mei-all</emph>, but extends it even
```

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -787,28 +782,28 @@
                      will be allowed, so that the absence of a specific information can not be
                      misunderstood as an oversight of the encoders. Larger editorial projects like
-                     <hi rend="italic">Complete Works</hi> editions typically use <hi rend="italic">Editorial Guidelines</hi> (german:
+                     <emph>Complete Works</emph> editions typically use <term>Editorial Guidelines</term> (german:
```
```suggestion
                     <emph>Complete Works</emph> editions typically use <emph>Editorial Guidelines</emph> (german:
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: Benjamin W. Bohl <b.w.bohl@gmail.com>
Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -787,28 +782,28 @@
+                        the TEI</ref>, and the <title><ref
                         target="https://tei-c.org/guidelines/customization/getting-started-with-p5-odds/"
-                        >Getting Started with P5 ODDs</ref>" document.</p>
+                        >Getting Started with P5 ODDs</ref></title> document.</p>
```
```suggestion
                        >Getting Started with P5 ODDs</ref> document.</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/01-introduction.xml:
```diff
@@ -461,45 +461,44 @@
+                     allowed in music. In 1965, Milton Babbitt distinguished between
+                     <term>graphemic</term>, <term>acoustic</term> and <term>auditory</term>
+                     aspects of music (Babbitt, Milton: <title>The Use of Computers in Musicological Research</title>,
+                     in: <title>Perspectives of New Music</title> 3/2 (1965), p. 76).</p>
```
```suggestion
                     in: <hi rend="italic">Perspectives of New Music</hi> 3/2 (1965), p. 76).</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/02-shared.xml:
```diff
@@ -590,14 +590,14 @@
                      </figure>
                   </p>
                   <p>When a reference to an external entity is not a complete URI it is resolved against the current base URI; if not defined by other means this would be the location of the current document. The above example consequently would mean, that the file `myImage.jpg` referenced from <gi scheme="MEI">graphic</gi> resides at the same location (in the same folder) as the MEI-file.</p>
```
```suggestion
                  <p>When a reference to an external entity is not a complete URI it is resolved against the current base URI; if not defined by other means this would be the location of the current document. The above example consequently would mean, that the file <code>myImage.jpg</code> referenced from <gi scheme="MEI">graphic</gi> resides at the same location (in the same folder) as the MEI-file.</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/02-shared.xml:
```diff
@@ -590,14 +590,14 @@
                   </p>
                   <p>When a reference to an external entity is not a complete URI it is resolved against the current base URI; if not defined by other means this would be the location of the current document. The above example consequently would mean, that the file `myImage.jpg` referenced from <gi scheme="MEI">graphic</gi> resides at the same location (in the same folder) as the MEI-file.</p>
-                  <p>The <att>xml:base</att> attribute may be used “to specify a base URI other than the base URI of the document or external entity.” (Marsch, Jonathan; Tobin, Richard: XML Base (Second Edition). W3C Recommendation 28 January 2009. online at: <ref target="http://www.w3.org/TR/2009/REC-xmlbase-20090128/">http://www.w3.org/TR/2009/REC-xmlbase-20090128/</ref>).</p>
+                  <p>The <att>xml:base</att> attribute may be used <quote>to specify a base URI other than the base URI of the document or external entity</quote>. (Marsch, Jonathan; Tobin, Richard: XML Base (Second Edition). W3C Recommendation 28 January 2009. online at: <ref target="http://www.w3.org/TR/2009/REC-xmlbase-20090128/">http://www.w3.org/TR/2009/REC-xmlbase-20090128/</ref>).</p>
```
```suggestion
                  <p>The <att>xml:base</att> attribute may be used <quote>to specify a base URI other than the base URI of the document or external entity.</quote> (Marsch, Jonathan; Tobin, Richard: XML Base (Second Edition). W3C Recommendation 28 January 2009. online at: <ref target="http://www.w3.org/TR/2009/REC-xmlbase-20090128/">http://www.w3.org/TR/2009/REC-xmlbase-20090128/</ref>).</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/03-metadata.xml:
```diff
@@ -477,7 +477,7 @@
                   <div xml:id="headerNotesStatement" type="div4">
                      <head>Notes Statement</head>
-                     <p>The <gi scheme="MEI">notesStmt</gi> element is the sixth component of the <gi scheme="MEI">fileDesc</gi> element and is optional. If used, it contains one or more <gi scheme="MEI">annot</gi> elements, each containing a single piece of descriptive information of the kind treated as ‘general notes’ in traditional bibliographic descriptions.</p>
+                     <p>The <gi scheme="MEI">notesStmt</gi> element is the sixth component of the <gi scheme="MEI">fileDesc</gi> element and is optional. If used, it contains one or more <gi scheme="MEI">annot</gi> elements, each containing a single piece of descriptive information of the kind treated as <hi rend="italic">general notes</hi> in traditional bibliographic descriptions.</p>
```
```suggestion
                     <p>The <gi scheme="MEI">notesStmt</gi> element is the sixth component of the <gi scheme="MEI">fileDesc</gi> element and is optional. If used, it contains one or more <gi scheme="MEI">annot</gi> elements, each containing a single piece of descriptive information of the kind treated as <q>general notes</q> in traditional bibliographic descriptions.</p>
```

krHERO commented on source/docs/03-metadata.xml:
```diff
@@ -1048,7 +1048,7 @@
                   <p>The <gi scheme="MEI">creation</gi> element is intended to contain a brief, machine-processable statement of the circumstances of the work’s creation. Its content is limited to text and the <gi scheme="MEI">date</gi> and <gi scheme="MEI">geogName</gi> elements.</p>
                   <p>The <gi scheme="MEI">history</gi> element is a container for additional non-bibliographic details relating to a work. It may use the <gi scheme="MEI">eventList</gi> element to provide a list of key events in the creation and performance history of the work. The <gi scheme="MEI">eventList</gi> element is comprised of <gi scheme="MEI">event</gi> elements containing a brief desc ...
-                  <p>Event lists and other text components, such as paragraphs, tables, lists, and text divisions (<gi scheme="MEI">div</gi>) may be interleaved when an 'essay-like' work history is desired.</p>
+                  <p>Event lists and other text components, such as paragraphs, tables, lists, and text divisions (<gi scheme="MEI">div</gi>) may be interleaved when an <hi rend="italic">essay-like</hi> work history is desired.</p>
```
```suggestion
                  <p>Event lists and other text components, such as paragraphs, tables, lists, and text divisions (<gi scheme="MEI">div</gi>) may be interleaved when an <q>essay-like</q> work history is desired.</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/03-metadata.xml:
```diff
@@ -1680,7 +1680,7 @@
                            </specList>
                         </p>
-                        <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (e.g., <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right  ...
+                        <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (e.g., <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right  ...
```
```suggestion
                        <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (e.g., <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a <emph>gap</emph> in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it’s own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/04-cmn.xml:
```diff
@@ -979,7 +979,7 @@
                   <div xml:id="cmnFermata" type="div4">
                      <head>Fermata</head>
-                     <p>A very common feature of music notation from the CMN period is the so-called ‘fermata’ (or ‘corona’ in Italian). It is usually written as a dot above or below an arc. It may stand above or below the staff it affects with its ‘open’ side usually facing towards the staff. A fermata indicates that a related note or rest should be held longer than its written duration would no ...
+                     <p>A very common feature of music notation from the CMN period is the so-called <hi rend="italic">fermata</hi> (or <hi rend="italic">corona</hi> in Italian). It is usually written as a dot above or below an arc. It may stand above or below the staff it affects with its <hi rend="italic">open</hi> side usually facing towards the staff. A fermata indicates that a related note o ...
```
```suggestion
                     <p>A very common feature of music notation from the CMN period is the so-called <hi rend="italic">fermata</hi> (or <hi rend="italic">corona</hi> in Italian). It is usually written as a dot above or below an arc. It may stand above or below the staff it affects with its <q>open</q> side usually facing towards the staff. A fermata indicates that a related note or rest should be held longer than its written duration would normally require. Sometimes, a fermata occurs over or under a bar line to indicate a pause or even the end of a phrase or section.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -1087,21 +1087,21 @@
                   <head>Manuscripts</head>
-                  <p>Samples of MEI of St Gall notation are taken from the winter volume of the so-called ”Hartker Antiphonary” <ref target="https://www.e-codices.unifr.ch/en/list/one/csg/0359">CH-SGs Cod. Sang. 390</ref>.</p>
-                  <p>Samples of MEI of Old Hispanic notation are taken from the ”León Antiphoner” <ref target="https://bvpb.mcu.es/es/catalogo_imagenes/grupo.cmd?path=26408">E-L MS 8</ref>.</p>
+                  <p>Samples of MEI of St Gall notation are taken from the winter volume of the so-called <title>Hartker Antiphonary</title> <ref target="https://www.e-codices.unifr.ch/en/list/one/csg/0359">CH-SGs Cod. Sang. 390</ref>.</p>
```
```suggestion
                  <p>Samples of MEI of St Gall notation are taken from the winter volume of the so-called <q>Hartker Antiphonary</q> <ref target="https://www.e-codices.unifr.ch/en/list/one/csg/0359">CH-SGs Cod. Sang. 390</ref>.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -1087,21 +1087,21 @@
                <head>Bibliographic References</head>
-               <p>Bain, Jennifer, Inga Behrendt, and Kate Helsen. 2014. “Linienlose Neumen und ihre Repräsentation mit MEI Schema, Herausforderungen in der Arbeit im Optical Neume Recognition Project (ONRP).” <hi rend="italic">Digitale Rekonstruktionen mittelalterlicher Bibliotheken</hi>. Edited by Sabine Philippi and Philipp Vanscheidt. <hi rend="italic">Trierer Beiträge zu den historischen Kult ...
-               <p>Behrendt, Inga, Jennifer Bain, and Kate Helsen. 2017. “MEI Kodierung der frühesten Notation in linienlosen Neumen.” <hi rend="italic">Kodikologie und Paläographie im Digitalen Zeitalter 4 / Codicology and Palaeography in the Digital Age</hi>. Vol. 4. Edited by Hannah Busch, Franz Fischer, and Patrick Sahle, with the cooperation of Philip Hegel and Celiz Krause, Norderstedt 2016. ...
+               <p>Bain, Jennifer, Inga Behrendt, and Kate Helsen. 2014. <title>Linienlose Neumen und ihre Repräsentation mit MEI Schema, Herausforderungen in der Arbeit im Optical Neume Recognition Project (ONRP).</title> <hi rend="italic">Digitale Rekonstruktionen mittelalterlicher Bibliotheken</hi>. Edited by Sabine Philippi and Philipp Vanscheidt. <hi rend="italic">Trierer Beiträge zu den hist ...
```
```suggestion
               <p>Bain, Jennifer, Inga Behrendt, and Kate Helsen. 2014. <q>Linienlose Neumen und ihre Repräsentation mit MEI Schema, Herausforderungen in der Arbeit im Optical Neume Recognition Project (ONRP).</q> <hi rend="italic">Digitale Rekonstruktionen mittelalterlicher Bibliotheken</hi>. Edited by Sabine Philippi and Philipp Vanscheidt. <hi rend="italic">Trierer Beiträge zu den historischen Kulturwissenschaften</hi> 12: 119–32.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -1087,21 +1087,21 @@
+               <p>Behrendt, Inga, Jennifer Bain, and Kate Helsen. 2017. <title>MEI Kodierung der frühesten Notation in linienlosen Neumen.</title> <hi rend="italic">Kodikologie und Paläographie im Digitalen Zeitalter 4 / Codicology and Palaeography in the Digital Age</hi>. Vol. 4. Edited by Hannah Busch, Franz Fischer, and Patrick Sahle, with the cooperation of Philip Hegel and Celiz Krause, Nord ...
                <p>
-                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. 2019. <ref target="https://music-encoding.org/conference/2019/abstracts_mec2019/MEI%202019_De%20Luca%20et%20alii.pdf">”Cantus Ultimus’ MEI Neume Module and its Interoperability Across Ch ...
+                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. 2019. <ref target="https://music-encoding.org/conference/2019/abstracts_mec2019/MEI%202019_De%20Luca%20et%20alii.pdf"><title>Cantus Ultimus’ MEI Neume Module and its Interoperability Acr ...
```
```suggestion
                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. 2019. <ref target="https://music-encoding.org/conference/2019/abstracts_mec2019/MEI%202019_De%20Luca%20et%20alii.pdf"><q>Cantus Ultimus’ MEI Neume Module and its Interoperability Across Chant Notations</q></ref>. Music Encoding Conference, Vienna.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -1087,21 +1087,21 @@
-                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. “Capturing Early Notations in MEI: The Case of Old Hispanic Neumes”. <hi rend="italic">Musiktheorie-Zeitschrift für Musikwissenschaft 2</hi>, 2019: 229-49.</p>
-               <p>Helsen, Kate, Inga Behrendt, and Jennifer Bain. 2017. “<ref target="https://hrcak.srce.hr/index.php?show=clanak&amp;id_clanak_jezik=284211">A Morphology of Medieval Notations in the Optical Neume Recognition Project</ref>.” <hi rend="italic">Arti musices: Croatian Musicological Review</hi> 48/2: 241–266.</p>
-               <p>MEI Guidelines v4, ch. 6: <ref target="https://music-encoding.org/guidelines/v4/content/neumes.html">Neume Notation</ref> introducing <gi scheme="MEI">nc</gi> as “neume component”.</p>
+                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. <title>Capturing Early Notations in MEI: The Case of Old Hispanic Neumes</title>. <hi rend="italic">Musiktheorie-Zeitschrift für Musikwissenschaft 2</hi>, 2019: 229-49.</p>
```
```suggestion
                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. <q>Capturing Early Notations in MEI: The Case of Old Hispanic Neumes</q>. <hi rend="italic">Musiktheorie-Zeitschrift für Musikwissenschaft 2</hi>, 2019: 229-49.</p>
```

krHERO commented on source/docs/06-neumes.xml:
```diff
@@ -1087,21 +1087,21 @@
-               <p>Helsen, Kate, Inga Behrendt, and Jennifer Bain. 2017. “<ref target="https://hrcak.srce.hr/index.php?show=clanak&amp;id_clanak_jezik=284211">A Morphology of Medieval Notations in the Optical Neume Recognition Project</ref>.” <hi rend="italic">Arti musices: Croatian Musicological Review</hi> 48/2: 241–266.</p>
-               <p>MEI Guidelines v4, ch. 6: <ref target="https://music-encoding.org/guidelines/v4/content/neumes.html">Neume Notation</ref> introducing <gi scheme="MEI">nc</gi> as “neume component”.</p>
+                  <ref target="https://sites.google.com/fcsh.unl.pt/elsadeluca/">De Luca, Elsa</ref>, Jennifer Bain, Inga Behrendt, Ichiro Fujinaga, Kate Helsen, Alessandra Ignesti, Debra Lacoste, and Sarah Long. <title>Capturing Early Notations in MEI: The Case of Old Hispanic Neumes</title>. <hi rend="italic">Musiktheorie-Zeitschrift für Musikwissenschaft 2</hi>, 2019: 229-49.</p>
+               <p>Helsen, Kate, Inga Behrendt, and Jennifer Bain. 2017. <title><ref target="https://hrcak.srce.hr/index.php?show=clanak&amp;id_clanak_jezik=284211">A Morphology of Medieval Notations in the Optical Neume Recognition Project</ref>.</title> <hi rend="italic">Arti musices: Croatian Musicological Review</hi> 48/2: 241–266.</p>
```
```suggestion
               <p>Helsen, Kate, Inga Behrendt, and Jennifer Bain. 2017. <q><ref target="https://hrcak.srce.hr/index.php?show=clanak&amp;id_clanak_jezik=284211">A Morphology of Medieval Notations in the Optical Neume Recognition Project</ref>.</q> <hi rend="italic">Arti musices: Croatian Musicological Review</hi> 48/2: 241–266.</p>
```

krHERO commented on source/docs/09-textencoding.xml:
```diff
@@ -868,7 +868,7 @@
                            concept, both consistent of and denoting generalities; he or
                            she groups examples of music according to similarities
-                           between them.” (Source: “Style”, Grove Music Online,
+                           between them.</quote> (Source: <title>Style</title>, Grove Music Online,
```
```suggestion
                           between them.</quote> (Source: <q>Style</q>, <hi rend="italic">Grove Music Online</hi>,
```

krHERO commented on source/docs/09-textencoding.xml:
```diff
@@ -1016,13 +1016,13 @@
                         </figure>
                      </p>
-                     <p>In this example, the nested <gi scheme="MEI">relatedItem</gi> / <gi scheme="MEI">bibl</gi> provides information about the ‘container’ where the outer <gi scheme="MEI">bibl</gi> may be found. The kind of relation is expressed using the <att>rel</att> attribute. It describes the relationship of the child <gi scheme="MEI">bibl</gi> to the <gi scheme="MEI">relatedItem</gi>’s p ...
+                     <p>In this example, the nested <gi scheme="MEI">relatedItem</gi> / <gi scheme="MEI">bibl</gi> provides information about the <hi rend="italic">container</hi> where the outer <gi scheme="MEI">bibl</gi> may be found. The kind of relation is expressed using the <att>rel</att> attribute. It describes the relationship of the child <gi scheme="MEI">bibl</gi> to the <gi scheme="MEI" ...
```
```suggestion
                     <p>In this example, the nested <gi scheme="MEI">relatedItem</gi> / <gi scheme="MEI">bibl</gi> provides information about the <q>container</q> where the outer <gi scheme="MEI">bibl</gi> may be found. The kind of relation is expressed using the <att>rel</att> attribute. It describes the relationship of the child <gi scheme="MEI">bibl</gi> to the <gi scheme="MEI">relatedItem</gi>’s parent <gi scheme="MEI">bibl</gi>.</p>
```

krHERO commented on source/docs/11-scholarlyediting.xml:
```diff
@@ -275,7 +275,7 @@
                         </specList>
                      </p>
-                     <p>Like any other ‘controlEvent’ (see <ptr target="#eventsControlevents"/>), <gi scheme="MEI">cpMark</gi> is placed in the score using the <att>staff</att> and <att>tstamp</att> attributes. The end point of the mark itself, when necessary, may be indicated using the <att>tstamp2</att> attribute. The source material, which is intended to be inserted in the space indicated by t ...
+                     <p>Like any other <hi rend="italic">controlEvent</hi> (see <ptr target="#eventsControlevents"/>), <gi scheme="MEI">cpMark</gi> is placed in the score using the <att>staff</att> and <att>tstamp</att> attributes. The end point of the mark itself, when necessary, may be indicated using the <att>tstamp2</att> attribute. The source material, which is intended to be inserted in the ...
```
```suggestion
                     <p>Like any other <emph>controlEvent</emph> (see <ptr target="#eventsControlevents"/>), <gi scheme="MEI">cpMark</gi> is placed in the score using the <att>staff</att> and <att>tstamp</att> attributes. The end point of the mark itself, when necessary, may be indicated using the <att>tstamp2</att> attribute. The source material, which is intended to be inserted in the space indicated by the copy mark, can be identified by the attributes <att>origin.tstamp</att>, <att>origin.tstamp2</att>, <att>origin.staff</att> and <att>origin.layer</att>. While <att>origin.tstamp</att> provides the relative distance from the beginning of the <q>gap</q>, <att>origin.tstamp</att> is relative to the position identified by <att>origin.tstamp</att>. However, <att>origin.tstamp</att> defaults to the same value as <att>tstamp2</att> and should only be provided when necessary. When neither <att>origin.staff</att> nor <att>origin.tstamp</att> are provided, they take the same values as the cpMark’s <att>staff</att> and <att>tstamp</att> attributes; that is, they indicate a strict <hi rend="italic">vertical</hi> or <hi rend="italic">horizontal</hi> copy.</p>
```

krHERO commented on source/docs/12-facsimilesperformances.xml:
```diff
@@ -251,7 +251,7 @@
                      </figure>
                   </p>
-                  <p>Clips can also be aligned with components of the musical text encoded in the <gi scheme="MEI">body</gi>. The <att>startid</att> attribute can be used to specify the starting element in the sequence of events to which the clip corresponds. The following example shows the use of clip elements to identify the exposition of the first movement from Beethoven’s piano sonata Op. 14, ...
+                  <p>Clips can also be aligned with components of the musical text encoded in the <gi scheme="MEI">body</gi>. The <att>startid</att> attribute can be used to specify the starting element in the sequence of events to which the clip corresponds. The following example shows the use of clip elements to identify the exposition of the first movement from Beethoven’s piano sonata Op. 14, ...
```
```suggestion
                  <p>Clips can also be aligned with components of the musical text encoded in the <gi scheme="MEI">body</gi>. The <att>startid</att> attribute can be used to specify the starting element in the sequence of events to which the clip corresponds. The following example shows the use of clip elements to identify the exposition of the first movement from Beethoven’s piano sonata Op. 14, no. 2 and its concluding <hi rend="italic">codetta</hi>.</p>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen commented on source/docs/05-mensural.xml:
```diff
@@ -264,16 +264,19 @@
+                      <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/mensuration_changes.mei" parse="text"/></egXML>
+                    </figure></p>
+                  <p><hi rend="bold">Sesquialtera</hi> is frequently used to change the mensuration. The effect of the sesquialtera on the mensuration can be encoded by using the <att>tempus</att> and <att>prolatio</att> attributes of <gi scheme="MEI">mensur</gi> (for example, when changing the tempus to <hi rend="italic">perfect</hi>, the effect can be encoded in <att>tempus</att>=<val>3</val>). ...
+                    <val>2</val>, and <att>level</att> to define the note level the sesquialtera is applied to (<abbr>e.g.</abbr>, <att>level</att>=<val>semibrevis</val>).
```
```suggestion
                      <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/mensuration_changes.mei" parse="text"/></egXML>
                    </figure>
                  </p>
                  <p><hi rend="bold">Sesquialtera</hi> is frequently used to change the mensuration. The effect of the sesquialtera on the mensuration can be encoded by using the <att>tempus</att> and <att>prolatio</att> attributes of <gi scheme="MEI">mensur</gi> (for example, when changing the tempus to <hi rend="italic">perfect</hi>, the effect can be encoded in <att>tempus</att>=<val>3</val>). The actual sesquialtera can be encoded using <att>num</att>=<val>3</val>, <att>numbase</att>=<val>2</val>, and <att>level</att> to define the note level the sesquialtera is applied to (<abbr>e.g.</abbr>, <att>level</att>=<val>semibrevis</val>).
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen commented on source/docs/11-scholarlyediting.xml:
```diff
@@ -517,7 +517,7 @@
                <div xml:id="geneticStates" type="div3">
                   <head>Encoding Genetic States</head>
-                  <p>Leaving aside temporary breaks, a compositional process is continuous: the composer’s writing operations have happened in a strict order, which could be specified if his working would have been filmed. However, this exact order is rarely ever recoverable from the surviving manuscripts, prints or other materials. Instead, relative statements can be made – the red pencil must h ...
+                  <p>Leaving aside temporary breaks, a compositional process is continuous: the composer’s writing operations have happened in a strict order, which could be specified if his working would have been filmed. However, this exact order is rarely ever recoverable from the surviving manuscripts, prints or other materials. Instead, relative statements can be made – the red pencil must h ...
```
their?

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

krHERO commented on source/docs/05-mensural.xml:
```diff
@@ -161,9 +161,9 @@
+                     <item>One of the three categories of a longa in Ars antiqua (<val>duplex</val>, <val>perfecta</val>, and <val>imperfecta</val>)</item>
                   </list>
-                  <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the ‘voice’ staff renders  ...
+                  <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the <val>voice</val> staff ...
```
```suggestion
                  <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the <emph>voice</emph> staff renders the notes in the code snippet, and the <emph>reference</emph> staff, together with the dotted bar lines, are shown to help to visualize the relative values of the notes in the <emph>voice</emph> staff.</p>
```

krHERO commented on source/docs/07-tablature.xml:
```diff
@@ -188,7 +188,7 @@
-                  <p>Unlike other types of lute tablature, German lute tablature encodes the course and the fret information simultaneously, and in that sense is different from the staff-based (<abbr>i.e.</abbr>, Italian and French) lute tablature types. Vertical positioning thus does not correspond to the course to be played, and may instead reflect aesthetic or layout conventions or, in some ca ...
+                  <p>Unlike other types of lute tablature, German lute tablature encodes the course and the fret information simultaneously, and in that sense is different from the staff-based (<abbr>i.e.</abbr>, Italian and French) lute tablature types. Vertical positioning thus does not correspond to the course to be played, and may instead reflect aesthetic or layout conventions or, in some ca ...
                   <list rend="bulleted">
                      <item>The <att>tab.align</att> attribute on <gi scheme="MEI">staffDef</gi> specifies the vertical alignment of the tablature notes. It typically takes two of the four data.VERTICALALIGNMENT values (<val>top</val> and <val>bottom</val>), and can be used to encode a consistent alignment of the tablature notes towards the top row or the bottom row. As with <att>valign</att>, <va ...
```
```suggestion
                     <item>The <att>tab.align</att> attribute on <gi scheme="MEI">staffDef</gi> specifies the vertical alignment of the tablature notes. It typically takes two of the four data.VERTICALALIGNMENT values (<val>top</val> and <val>bottom</val>), and can be used to encode a consistent alignment of the tablature notes towards the top row or the bottom row. As with <att>valign</att>, <val>top</val> is the default value for <att>tab.align</att>. Therefore, in practice, <att>tab.align</att> is only required in the <q>bottom</q> scenario.</item>
```

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestions from code review

Co-authored-by: krHERO <krHERO@users.noreply.github.com>

musicEnfanthen committed:
Apply suggestion from @musicEnfanthen

musicEnfanthen committed:
Merge branch 'develop' into quotation-marks-1498

bwbohl reviewed (approved):
The encoding errors I encountered were due to a sync issue on my side. So I consider this a go

bwbohl committed:
Merge branch 'develop' into quotation-marks-1498

rettinghaus reviewed (changes requested):
Looks good. One last wish.

bwbohl commented on source/docs/01-introduction.xml:
```diff
@@ -459,45 +459,44 @@
+                     allowed in music. In 1965, Milton Babbitt distinguished between
+                     <emph>graphemic</emph>, <emph>acoustic</emph> and <emph>auditory</emph>
+                     aspects of music (Babbitt, Milton: <q>The Use of Computers in Musicological Research</q>,
+                     in: <hi rend="italic">Perspectives of New Music</hi> 3/2 (1965), p. 76).</p>
```
@rettinghaus, completely with you, but let’s just tackle that in another iteration through the sources ;-)
